### DISCUSSION TITLE
vpcLookup on VPC that is shared (viaRAM) across accounts.

### DISCUSSION THREAD

**Discussion by mrpackethead** [aws/aws-cdk#24608] [Q&A]
When subnets are shared from one account to another using RAMs, the Tags dont' get shared at the same time.   This makes importing a vpc  using the fromLookup() less functional than desirable.    In particular it makes it impossible to specify subnets using the subnetGroupName.  

I'm wondering if there is a sensible way of being able to achieve this kind of functionality...   Perhaps creating another contextProvidert that could work across account?

```
const attributes: cxapi.VpcContextResponse = ContextProvider.getValue(scope, {
      provider: cxschema.ContextProvider.VPC_PROVIDER,
      props: {
        ...overrides,
        filter,
        returnAsymmetricSubnets: true,
        subnetGroupNameTag: options.subnetGroupNameTag,
      } as cxschema.VpcContextQuery,
      dummyValue: undefined,
    }).value;
```

**Comment by vumdao** [comment]
I'm also looking for solution of this
